sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2305,6 +2451,20 @@ public partial class SenderAuthorization
         public string Role { get { throw null; } }
         public string Scope { get { throw null; } }
     }
+    public partial class StorageBlobDestination
+    {
+        public StorageBlobDestination() { }
+        public string ContainerName { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        public string StorageAccountResourceId { get { throw null; } set { } }
```

Comment:
Is this property an arm resource id? If so, please change the format to `arm-id`

Reply:
Where/How do I do that?

Reply:
Done